docs/components/InputFile/InputFile.stories.mdx

```diff
@@ -0,0 +1,60 @@
+import { Meta } from "@storybook/addon-docs";
+
+# Input File (Web Only)
```

Comment:
I don't think we need to add this (Web Only), as it just have web stories, I think it's already implicit